ios/CameraView+TakePhoto.swift

```diff
@@ -54,29 +54,64 @@ extension CameraView {
       }
 
       // Create photo settings
-      let photoSettings = AVCapturePhotoSettings(format: format)
+      var photoSettings: AVCapturePhotoSettings?
+
+      if #available(iOS 14.3, *), self.enableRawCapture?.boolValue == true {
+        let query =
+          photoOutput.isAppleProRAWEnabled ? { AVCapturePhotoOutput.isAppleProRAWPixelFormat($0) } : { AVCapturePhotoOutput.isBayerRAWPixelFormat($0) }
+
+        // Retrieve the RAW format, favoring Apple ProRAW when it's in an enabled state.
+        guard let rawFormat =
+          photoOutput.availableRawPhotoPixelFormatTypes.first(where: query) else {
+          fatalError("No RAW format found.")
+        }
+
+        let processedFormat = [AVVideoCodecKey: AVVideoCodecType.hevc]
+
+        photoSettings = AVCapturePhotoSettings(rawPixelFormatType: rawFormat,
+                                               processedFormat: processedFormat)
+      } else {
+        photoSettings = AVCapturePhotoSettings(format: format)
+      }
```

Comment:
Could we maybe wrap that in a separate class? I don't want to cloud up this file with such nested and very specific code. 
Additionally, `photoSettings` is now optional and we're force unwrapping it down there, so this is prone to null unwraps.